v21.1/set-locality.md

```diff
@@ -65,7 +65,7 @@ For more information about how table localities work, see [Regional tables](mult
 ### Set the table locality to `REGIONAL BY ROW`
 
 {{site.data.alerts.callout_info}}
-[Changefeeds](stream-data-out-of-cockroachdb-using-changefeeds.html) are not currently supported on regional by row tables. When a table's locality is set to `REGIONAL BY ROW`, any changefeed jobs targeting that that table will fail.
+Changefeeds are supported on regional by row tables from v21.1 onward. Before setting the locality to `REGIONAL BY ROW` on a table targeted by a changefeed, read the considerations in [Changefeeds on regional by row tables](stream-data-out-of-cockroachdb-using-changefeeds.html#changefeeds-on-regional-by-row-tables).
```

Comment:
I think the backport to 21.1 hasn't quite got in yet, so we might want to wait a couple weeks before merging this in